src/chapters/http/http-responses.rst

```diff
@@ -0,0 +1,163 @@
+Responses
+=========
+
+.. index::
+   single: HTTP; response
+
+Every time a web server receives an HTTP request, it sends an HTTP response
+back to the client.
+
+A generic HTTP response looks like this:
+
+::
+
+   HTTP/2.0 200 OK
+   Date: Wed, 22 May 2020 17:36:50 GMT
+   Content-Type: text/html; charset=UTF-8
+   Content-Length: 8050
+   Last-Modified: Wed, 22 May 2020 17:33:45 GMT
+
+   <!DOCTYPE html>
+   <html>
+   <head>
+   <meta charset="UTF-8">
+   <!--Rest of HTML page -->
+   </html>
+
+The structure has these components:
+
+.. index::
+   single: HTTP; response code
+   single: HTTP; response headers
+   single: HTTP; response line
+
+#. **Response line:** The first line of the response contains status
+   information like the **response code**. In this example, the response code
+   is ``200``, which indicates the request was successfully fulfilled.
+#. **Response headers:** Below the response line are the **response headers**.
+   Similar to request headers, these are key-value pairs that contain data
+   about the response. In the example above, these include information about
+   the date and time, the data being sent (text and HTML), the size of the
+   response body, and when it was last modified.
+#. **Blank line:** This marks the end of the response headers.
+#. **Response body (Optional):** Below the blank line, the request body takes
```

Comment:
typo: "the request body..." -> "the response body..."

Reply:
Fixed.